test/system/helpers.bash

```diff
@@ -383,6 +383,13 @@ function journald_unavailable() {
     return 1
 }
 
+# Returns the name of the local pause image.
+function pause_image() {
+    run_podman version --format "{{.Server.Version}}-{{.Server.Built}}" > /dev/null
+    local name=$(basename ${output:-[null]})
```

Comment:
This looks like a NOP, but there might be circumstances in which the version includes a slash. If there are, could you add a comment explaining what that's all about? Or if it truly is a NOP, could you remove this line?

(also: if you need to keep this line, could you rename the variable to `tag`, which seems more accurate?)

Reply:
Great catch!